pycrostates/io/ch_data.py

```diff
@@ -48,6 +48,8 @@ def __init__(self, data: NDArray[float], info: Union[Info, CHInfo]):
             )
         self._data = data
         self._info = info if isinstance(info, ChInfo) else ChInfo(info)
+        # Preload
+        self.preload = True
```

Comment:
Let's define it as a property, at least this way it can not be modified.

```
@property
def preload(self):
    """Preload required by some MNE functions."""
    return True